viewcontroller/viewcontroller.go

```diff
@@ -108,48 +100,61 @@ func NewRolloutViewController(rolloutId string, noColor bool) *RolloutViewContro
 	}
 }
 
-func (c *RolloutViewController) GetRollout() (rollout.DetailedRollout, error) {
-	detailedRollout := rollout.DetailedRollout{}
+func (c *RolloutViewController) GetRollout() (*rollout.DetailedRollout, error) {
+	detailedRollout := &rollout.DetailedRollout{}
+	errors := make(chan error)
+	wg := &sync.WaitGroup{}
+	wg.Add(3)
 
 	if c.previousRollout == nil {
-		strategy, err := c.rolloutRepository.GetStrategy(c.rolloutId)
-		if err != nil {
-			return detailedRollout, err
-		}
-		detailedRollout.Definition.Strategy = strategy
+		wg.Add(1)
+		go func(wg *sync.WaitGroup, errors chan error) {
+			defer wg.Done()
+			strategy, err := c.rolloutRepository.GetStrategy(c.rolloutId)
```

Comment:
add an empty line, please format the code with empty lines
you know where I like them

Reply:
please specify places, I remember only about separating if each from other